packages/contracts-bedrock/foundry.toml

```diff
@@ -26,7 +26,7 @@ bytecode_hash = 'none'
 build_info_path = 'artifacts/build-info'
 ast = true
 evm_version = "cancun"
-ignored_error_codes = ["transient-storage", "code-size", "init-code-size"]
+ignored_error_codes = [2394, "code-size", "init-code-size"]
```

Comment:
was this diff necessary?

Reply:
for some reason "transient-storage" doesn't seem to match the warning outputted by the compiler when using transient storage. looked around projects and tried `2394`, which worked right away. it seems that 2394 is more standardized.

Reply:
You can see [here in the foundry source](https://github.com/foundry-rs/foundry/blob/e0ea59cae26d945445d9cf21fdf22f4a18ac5bb2/crates/config/src/error.rs#L225) that it is supported. What was happening without it? It was emitting the warning? Are you sure you are on the right version of foundry?

Reply:
updated foundry and "transient-storage" works :)